internal/executor/instance/persistentworker/isolation/tart/tart.go

```diff
@@ -192,22 +188,47 @@ func (tart *Tart) Run(ctx context.Context, config *runconfig.RunConfig) (err err
 		break
 	}
 
+	tart.vm = vm
+
 	tart.logger.Debugf("IP %s retrieved from VM %s, running agent...", ip, vm.Ident())
 
 	bootLogger.Errorf("VM was assigned with %s IP", ip)
 	bootLogger.Finish(true)
 
+	return nil
+}
+
+func (tart *Tart) Run(ctx context.Context, config *runconfig.RunConfig) (err error) {
+	if tart.vm != nil && config.DirtyMode {
+		return fmt.Errorf("%w: dirty mode is not supported for a warmed instance", ErrFailed)
```

Comment:
Not all Tart instances will be standby instances (this assumption was also in https://github.com/cirruslabs/cirrus-cli/pull/698#discussion_r1523350920).

I think we still have a `cirrus run --dirty` (https://github.com/cirruslabs/cirrus-cli/issues/554) or are we dropping it for Tart?

Reply:
There is no dirty mode for persistent workers so this should never happen. Warmable instances won't be used for `cirrus run`